homeassistant/components/binary_sensor/axis.py

```diff
@@ -68,24 +82,33 @@ def is_on(self):
     @property
     def name(self):
         """Return the name of the event."""
-        return '{}_{}_{}'.format(
-            self.device_name, self.axis_event.event_type, self.axis_event.id)
+        return '{} {} {}'.format(
+            self.device.name, self.axis_event.event_type, self.axis_event.id)
+
+    @property
+    def unique_id(self):
```

Comment:
Please don't add more features than absolutely necessary per PR. This PR is about config entry support.

Reply:
Absolutely